app/src/main/java/com/evanfuhr/pokemondatabase/activities/MainActivity.java

```diff
@@ -1,26 +1,36 @@
 package com.evanfuhr.pokemondatabase.activities;
 
+import android.content.DialogInterface;
 import android.content.Intent;
+import android.content.SharedPreferences;
 import android.database.SQLException;
 import android.os.Bundle;
+import android.support.v7.app.AlertDialog;
 import android.support.v7.app.AppCompatActivity;
+import android.view.LayoutInflater;
 import android.view.Menu;
 import android.view.MenuItem;
 import android.view.View;
+import android.widget.AdapterView;
 import android.widget.Button;
-import android.widget.Toast;
+import android.widget.Spinner;
 
 import com.evanfuhr.pokemondatabase.R;
+import com.evanfuhr.pokemondatabase.adapters.VersionSpinnerAdapter;
 import com.evanfuhr.pokemondatabase.data.DataBaseHelper;
+import com.evanfuhr.pokemondatabase.data.VersionDAO;
+import com.evanfuhr.pokemondatabase.models.Version;
 
 import org.jetbrains.annotations.NonNls;
 
 import java.io.IOException;
+import java.util.List;
 
 public class MainActivity extends AppCompatActivity {
 
     @NonNls
     public static final String POKEMON = "Pokémon";
+    public static int _version_id = 0;
     @NonNls
```

Comment:
Do you need two `@NonNls` annotations? Can we group them somehow with line 31 (latest revision)?

Reply:
I'll see if that will work

Reply:
Turns out that annotation is only ever for the immediate following line. I just removed the both since that value wasn't even being used anymore.